biscuit-auth/src/token/builder.rs

```diff
@@ -1611,8 +1651,19 @@ impl From<biscuit_parser::builder::Rule> for Rule {
                     .map(|(k, v)| {
                         (
                             k,
-                            v.map(|bytes| {
-                                PublicKey::from_bytes(&bytes).expect("invalid public key")
+                            v.map(|pk| {
+                                PublicKey::from_bytes(
+                                    &pk.key,
+                                    match pk.algorithm {
+                                        biscuit_parser::builder::Algorithm::Ed25519 => {
```

Comment:
same as above

Reply:
done in https://github.com/biscuit-auth/biscuit-rust/commit/bdb8c7414107ddb16c8cd9b1bd6d4f0f0d0240f5